pkg/kubelet/cm/cpumanager/state/state_checkpoint.go

```diff
@@ -201,7 +201,7 @@ func (sc *stateCheckpoint) SetCPUSet(podUID string, containerName string, cset c
 	sc.cache.SetCPUSet(podUID, containerName, cset)
 	err := sc.storeState()
 	if err != nil {
-		klog.InfoS("Store state to checkpoint error", "err", err)
+		klog.ErrorS(err, "Store state to checkpoint error", "podUID", podUID, "containerName", containerName)
```

Comment:
Not directly related to this PR, but it looks a bit strange to me that `storeState` failures are not propagated up to the calling method. Any ideas why is that?